src/components/MonthSwitcher.vue

```diff
@@ -1,47 +1,50 @@
-<script setup>
-import { computed, reactive } from "vue";
-import { addMonths, subMonths, format } from "date-fns";
-import { mdiChevronLeft, mdiChevronRight } from "@mdi/js";
-const icons = reactive({
-  prev: mdiChevronLeft,
-  next: mdiChevronRight
-});
-const model = defineModel({ type: Date });
-function goPrev() {
-  model.value = subMonths(model.value, 1);
-}
-function goNext() {
-  model.value = addMonths(model.value, 1);
-}
+<script setup>  
+import { computed, reactive, onMounted } from "vue";  
+import { addMonths, subMonths, format } from "date-fns";  
+import { mdiChevronLeft, mdiChevronRight } from "@mdi/js";  
 
-const formattedDate = computed(() => {
-  return format(model.value, "MMMM yyyy");
-});
-</script>
+const icons = reactive({  
+  prev: mdiChevronLeft,  
+  next: mdiChevronRight  
+});  
 
-<template>
-  <div
-    style="
-      min-width: 300px;
-      min-height: 50px;
-      display: flex;
-      justify-content: center;
-    "
-  >
-    <v-btn
-      class="justify-center"
-      variant="text"
-      :icon="icons.prev"
-      @click="goPrev"
-    >
-    </v-btn>
-    <v-spacer />
-    <span style="align-self: center">
-      {{ formattedDate }}
-    </span>
-    <v-spacer />
-    <v-btn variant="text" :icon="icons.next" @click="goNext"></v-btn>
-  </div>
-</template>
+const model = reactive({ value: new Date() });  
+function goPrev() {  
+  model.value = subMonths(model.value, 1);  
+}  
+function goNext() {  
+  model.value = addMonths(model.value, 1);  
+}  
+
+const formattedDate = computed(() => {  
+  return format(model.value, "MMMM yyyy");  
+});  
 
-<style scoped lang="sass"></style>
+onMounted(() => {  
```

Comment:
This fixes the issue but is a triggered effect. I can see this causing problems in the future.

An easier, more obviouse and cleaner fix would be to change the initial Value [in this line](https://github.com/ClockGU/clock-supervisor-frontend/blob/e92f1cda16235895cd6c46cbecb24ace2c67d675/src/views/ContentView.vue#L11)